src/core/lib/gpr/alloc.cc

```diff
@@ -90,8 +90,7 @@ void* gpr_realloc(void* p, size_t size) {
   return p;
 }
 
-void* gpr_malloc_aligned(size_t size, size_t alignment_log) {
-  size_t alignment = ((size_t)1) << alignment_log;
+void* gpr_malloc_aligned(size_t size, size_t alignment) {
   size_t extra = alignment - 1 + sizeof(void*);
```

Comment:
Maybe add a `gpr_assert(((alignment - 1) & alignment) == 0)` to make sure we're being passed a power of two ?

Reply:
Good idea, done.